src/components/SigningOfficialDaaAgreementWrapper.js

```diff
@@ -98,10 +98,18 @@ export const SigningOfficialDaaAgreementWrapper = (props) => {
 // Wraps component and ensures that SO agrees to the
 // Broad and NIH agreements before proceeding to the given
 // component.
-export const ensureSoHasDaaAcknowledgement = (component, isDataSubmitterTab=false) => {
-  return (props) => h(SigningOfficialDaaAgreementWrapper, {isDataSubmitterTab}, [
-    h(component, props),
-  ]);
+export const ensureSoHasDaaAcknowledgement = (component, isLibraryCardIssueTable = false, isDataSubmitterTab = false) => {
+  return (props) => (
+    div({}, [
+      h(SigningOfficialDaaAgreementWrapper, { isDataSubmitterTab }, [
+        h(component, props),
+      ]),
+      isLibraryCardIssueTable && div({ style: { marginTop: '50px', background: '#eee', padding: '20px' } }, [
+        p({}, ['OMB No.: 0925-7775']),
+        p({}, ['Expiration Date: 06/30/2025']),
+        p({}, ['Public reporting burden for this collection of information is estimated to average 45 minutes per response, including the time for reviewing instructions, searching existing data sources, gathering and maintaining the data needed, and completing and reviewing the collection of information. An agency may not conduct or sponsor, and a person is not required to respond to, a collection of information unless it displays a currently valid OMB control number. Send comments regarding this burden estimate or any other aspect of this collection of information, including suggestions for reducing this burden to: NIH, Project Clearance Branch, 6705 Rockledge Drive, MSC 7974, Bethesda, MD 20892-7974, ATTN: PRA (0925-7775). Do not return the completed form to this address.']),
```

Comment:
It would be nice to extract this block into a separate component so it isn't copy/pasted in each component.